framework/src/mysql.js

```diff
@@ -378,7 +378,9 @@ const getTableInstance = async (tableName, tableConfig, connEndpoint = CONN_ENDP
 		if (params.distinct) {
 			query.countDistinct(`${params.distinct} as count`);
 		} else {
-			query.count(`${tableConfig.primaryKey} as count`);
+			const countColumnName = Array.isArray(tableConfig.primaryKey) ? tableConfig.primaryKey[0]
```

Comment:
TODO: Update all microservices after one round of review.

Reply:
We already have an issue to update `count/find` method to handle composite primary keys https://github.com/LiskHQ/lisk-service/issues/1188

Reply:
This was generating error as `stakes` table has composite primary key and is a small change, this is why doing it in this issue. Also, count does not depend on the columns unless we want to count distinct. In case of counting distinct, we are specifying one column anyway.